torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -776,6 +776,9 @@ class CudaKernelGenerator : private OptOutConstDispatch {
     // innermost ID of size 4 (float) or size 2 (double)?
     auto index = genTensorIndex(rop->getPhiloxIndex()->as<kir::TensorIndex>());
     int multiple = rop->dtype() == DataType::Double ? 2 : 4;
+    if (rop->getRNGOpType() == RNGOpType::NormalRange) {
+      multiple /= 2;
+    }
```

Comment:
curand box muller methods uses 4 unsinged int to generate a double random number and 2 unsigned int to generate a float normal random number, so I added a `multiple/2` here.

Reply:
Should we move this entire logic as a method of `RNGOp`? So in the codegen, you can just use `rop->getPhiloxMultiple()`